src/data/mcyMember.js

```diff
@@ -1,11 +1,10 @@
 import { nanoid } from "nanoid"
 
-export const mcyMember = [
+const mcyMembers = [
   {
     id: nanoid(10),
     cell: "대예배",
-    isChecked: false,
-    cellMember: [
+    checkedMember: [
```

Comment:
```suggestion
    checkedMember: [
```
여기 명칭이 checkedMember가 되는게 맞을지?
checked라고 하면 이미 출석 체크된 멤버라는 뜻이라고 이해되는데,
의도가 그게 맞는지 확인 필요

Reply:
@ykss 중석이가 출석 현황페이지에서 데이터 불러오는 방식을 위 처럼 변경해달라고 해서 의도된게 맞습니다!